content/sensu-core/2.0/reference/entities.md

```diff
@@ -14,21 +14,21 @@ menu:
 
 ### Why "Entities"?
 
-Sensu 2.0 uses an updated data model to address the latest paradigm shift in infrastructure and software delivery like containers and "serverless". Sensu is no longer confined to the concept of a client, we have "entities" that can can represent anything.
+Sensu 2.0 uses an updated data model to address the latest paradigm shift in infrastructure and software delivery like containers and "serverless", so we've adjusted the nomenclature accordingly. Sensu is no longer confined to the concept of a client, we have "entities" that can can represent anything.
 
 ### What is a Sensu entity?
 
-Entities, formally known as clients in Sensu 1.x, are monitoring agents (e.g. server, container, etc) that need to be monitored. They contain system information such as the hostname, OS, platform, and version. Entities not only provide context to event data but an event's uniqueness is determined by the check name and the ID of the entity upon which the check ran.
+An Entity, formally known as a client in Sensu 1.x, represents anything (e.g. server, container, network switch, etc) that needs to be monitored. It can contain system information such as the hostname, OS, platform, and version. An Entity not only provides context to event data (what/where the event is from) but an event's uniqueness is determined by the check name and the ID of the entity upon which the check ran.
```

Comment:
I like this a lot, especially if you adopt some form of the comment above that leaves the user with the concept of entities. 

Minor things: we will want to consistently capitalize keywords (Entity, Entities) or not. Right now we have a mix. Based on what I see so far, I'd recommend sticking with lowercase and maybe **bolding** new concepts when you introduce them (instead of capitalizing them).